src/main/java/spireMapOverhaul/zones/monsterZoo/BackgroundMonsterEffect.java

```diff
@@ -0,0 +1,373 @@
+package spireMapOverhaul.zones.monsterZoo;
+
+import basemod.ReflectionHacks;
+import basemod.abstracts.CustomMonster;
+import basemod.animations.AbstractAnimation;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.MathUtils;
+import com.badlogic.gdx.math.Vector2;
+import com.esotericsoftware.spine.Bone;
+import com.esotericsoftware.spine.Skeleton;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.monsters.beyond.*;
+import com.megacrit.cardcrawl.monsters.city.*;
+import com.megacrit.cardcrawl.monsters.exordium.*;
+import com.megacrit.cardcrawl.vfx.AbstractGameEffect;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+//Mostly copied from https://github.com/a-personal-account/PaleOfTheAncients/blob/master/src/main/java/paleoftheancients/finarubossu/vfx/BackgroundMonster.java
```

Comment:
Ah yes, I remember seeing this in the background of the final boss for Pale.

I skimmed through this class, not much need to review it in detail.